exporter/datasetexporter/README.md

```diff
@@ -34,13 +42,25 @@ If you do not want to specify `api_key` in the file, you can use the [builtin fu
   - `retry_max_elapsed_time` (default = 300s): Is the maximum amount of time spent trying to send a buffer.
 - `logs`:
   - `export_resource_info_on_event` (default = false): Include resource info to DataSet Event while exporting Logs. This is especially useful when reducing DataSet billable log volume.
+- `server_host`:
+  - `server_host` (default = ''): Use the specified value as server host for the event.
+  - `use_host_name` (default = true): Use the `hostname` of the node as server host used for the event.
```

Comment:
Same here.

Reply:
May want to be more explicit and say something along the lines of "Use hostname of the server where the collector is running for serverHost attribute for all the events processed by this collector".

Reply:
I have added section with examples.